src/main.js

```diff
@@ -313,6 +326,27 @@ ipcMain.on('download-url', (event, URL) => {
   });
 });
 
+let scheme;
+if (protocols && protocols[0] &&
```

Comment:
Should we consider the case where multiple protocols are listed?

Reply:
@yuya-oc I tried adding multiple protocols but after a while I realized that it's not possible because: `app.setAsDefaultProtocolClient(protocol)` just registers the last one.